data/json/items/melee.json

```diff
@@ -2007,5 +2063,159 @@
     "to_hit": 1,
     "category": "weapons",
     "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 8 ] ]
+  },
+  {
+    "id": "pike_fake",
+    "type": "GENERIC",
+    "category": "weapons",
+    "name": "awl pike",
+    "description": "This is a dull, cheaply made replica of a medieval weapon consisting of a wood shaft tipped with an iron spike.",
+    "weight": 2000,
+    "volume": 14,
+    "price": 4000,
+    "to_hit": 2,
+    "bashing": 10,
+    "cutting": 15,
+    "material": [ "aluminum", "wood" ],
+    "symbol": "/",
+    "color": "brown",
+    "looks_like": "pike",
+    "techniques": [ "IMPALE", "WBLOCK_2" ],
+    "qualities": [ [ "COOK", 1 ] ],
+    "flags": [ "SPEAR", "REACH_ATTACK", "REACH3", "NONCONDUCTIVE" ]
+  },
+  {
+    "id": "pike_inferior",
+    "type": "GENERIC",
+    "category": "weapons",
+    "name": "awl pike",
+    "description": "This is a medieval weapon consisting of a wood shaft tipped with an iron spike.  The spike seems to be pretty dull, and the whole thing feels poorly made.",
+    "weight": 6000,
+    "volume": 14,
+    "price": 40000,
+    "to_hit": 2,
+    "bashing": 12,
+    "cutting": 25,
+    "material": [ "budget_steel", "wood" ],
+    "symbol": "/",
+    "color": "brown",
+    "looks_like": "pike",
+    "techniques": [ "IMPALE", "WBLOCK_2" ],
+    "qualities": [ [ "COOK", 1 ] ],
+    "flags": [ "SPEAR", "REACH_ATTACK", "REACH3", "NONCONDUCTIVE" ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "mace_fake",
+    "name": "mace",
+    "description": "A light, cheaply made replica of a medieval weapon consisting of a wood handle with a heavy iron end.",
```

Comment:
It says it has an iron end, yet is made of aluminum?

Reply:
It says it's "a replica of" an iron weapon. This replica happens to be made of aluminum. Unclear? Yes. Technically correct English...? I think so.

Reply:
This description has been made clearer